downloader.sh

```diff
@@ -1,4 +1,10 @@
 a="$#"
+index(){
+	local link=$1
+	local x=$2
+	local start_end="$(echo $link | grep -aob '"' | grep -oE '[0-9]+' | sed "x;d")"
```

Comment:
Ok this is a  little bit hard I think (-.-)" Here is the function you need to write:-
```
index(){
	local start_end="$(echo $@ | grep -aob '"' | grep -oE '[0-9]+' | sed "${!#};d")"
	echo "$start_end"
}
```